src/views/SystemProfile/index.tsx

```diff
@@ -679,16 +676,13 @@ const SystemProfile = ({ id, modal }: SystemProfileProps) => {
                               systemId={systemId}
                             />
                           </div>
-                          {
-                            /* TODO: Make callout visible in EASI-2447 */
-                            // subinfo === 'team' && isMobile && (
-                            //   <EditPageCallout
-                            //     className="margin-top-4"
-                            //     // TODO: Get system modifiedAt value and add to props
-                            //     // modifiedAt={}
-                            //   />
-                            // )
-                          }
+                          {subinfo === 'team' && isMobile && (
+                            <EditPageCallout
+                              className="margin-top-4"
+                              // TODO: Get system modifiedAt value and add to props
+                              // modifiedAt={}
```

Comment:
@aterstriep Does this need to get implemented before we go live?

Reply:
Nope! Eventually the component will display the "last saved" time but it wasn't part of the AC for EASI-2439. I had started implementing it before I realized that, so I just commented it out for now.